src/iterative_ensemble_smoother/_iterative_ensemble_smoother.py

```diff
@@ -175,7 +234,7 @@ def update(self, param_ensemble: npt.NDArray[np.double]) -> npt.NDArray[np.doubl
             np.ix_(self.ensemble_mask, self.ensemble_mask)
         ]
         transition_matrix /= np.sqrt(ensemble_size - 1)
-        transition_matrix.flat[:: ensemble_size + 1] += 1.0
+        transition_matrix.flat[:: ensemble_size + 1] += 1
```

Comment:
Think I prefer 1.0 since it's more explicit.

Reply:
I used 1.0 since it works if the matrix is of type integer or float. If we use 1 then it will not work on integers. Not that the transition matrix is integer anyway :) can go back to 1.0 :)